content/20.pharmaceuticals.md

```diff
@@ -714,12 +714,27 @@ Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾) induce the expression
-SNG001 was already shown to be effective in reducing viral load in an _in vivo_ model of swine flu and _in vitro_ models of other coronavirus infections [@synairgen-SNG001].
+On March 18 2020, Synairgen plc received approval to start a phase II trial for SNG001, an IFN-𝛽-1a formulation to be delivered to the lungs via inhalation [@clinicaltrials:NCT04385095].
+SNG001, which contains recombinant interferon beta-1a, was already shown to be effective in reducing viral load in an _in vivo_ model of swine flu and _in vitro_ models of other coronavirus infections [@synairgen-SNG001].
 In July, a press release from Synairgen stated that SNG001 reduced progression to ventilation in a double-blind, placebo-controlled, multi-center study of 101 patients with an average age in the late 50s [@url:https://www.synairgen.com/wp-content/uploads/2020/07/200720-Synairgen-announces-positive-results-from-trial-of-SNG001-in-hospitalised-COVID-19-patients.pdf].
-They also reported that patients in the treatment group showed greater recovery and lower breathlessness.
-However, given that this information was released in a press release rather than in a manuscript and thus cannot be thoroughly reviewed, these findings should be considered preliminary.
-Additionally, the WHO Solidarity trial reported no significant effect of IFN-𝛽1a on patient survival during hospitalization [@doi:10.1056/NEJMoa2023184].
+These results were subsequently published in November 2020 [@doi:10/ghjzm4].
+The study reports that the participants were assigned at a ratio of 1:1 to receive either SNG001 or a placebo that lacked the active compound, by inhalation for up to 14 days.
+The primary outcome they assessed was the change in patients' score on the WHO Ordinal Scale for Clinical Improvement (OSCI) at day 15/16.
+SNG001 was associated with an odds ratio of 2.32 (95% CI 1.07 – 5.04, _p_ = 0.033) in the intention-to-treat analysis and 2.80 (95% CI 1.21 – 6.52, _p_ = 0.017) in the per-protocol analysis, corresponding to significant improvement in the SNG001 group on the OSCI at day 15/16.
+Some of the secondary endpoints analyzed also showed differences: at day 28, the OR for clinical improvement on the OSCI was 3.15 (95% CI 1.39 – 7.14, _p_ = 0.006), and the odds of recovery at day 15/16 and at day 28 were also significant between the two groups.
+Thus, this study suggested that IFN-𝛽1 administered via SNG001 may improve clinical outcomes.
+
+In contrast, the WHO Solidarity trial reported no significant effect of IFN-𝛽1a on patient survival during hospitalization [@doi:10.1056/NEJMoa2023184].
+An obvious difference between these two studies is their sample size, with the WHO Solidarity trial administering IFN-𝛽-1a to 2050 patients and comparing their response to 2050 controls.
+Here, the primary outcome analyzed was in-hospital mortality, and the rate ratio for the two groups was 1.16 (95% CI, 0.96 to 1.39; _p_ = 0.11).
```

Comment:
```suggestion
Here, the primary outcome analyzed was in-hospital mortality, and the rate ratio for the two groups was 1.16 (95% CI, 0.96 to 1.39; _p_ = 0.11) administering IFN-𝛽-1a to 2050 patients and comparing their response to 2050 controls..
```

Reply:
This is purely a stylistic change and up to you if you want to keep it. It felt a little confusing to me to talk about differences in setup between the two studies that don't necessarily contribute to the differences in the outcome. But then talk about the differences in setup that could lead to differences in outcome later.